test/resolver.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const tsConfigPaths = require('tsconfig-paths');
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'tsconfig-paths'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->